src/sentry_plugins/vsts/repository_provider.py

```diff
@@ -71,26 +71,58 @@ def create_repository(self, organization, data, actor=None):
     def delete_repository(self, repo, actor=None):
         pass
 
+    def transform_changes(self, patch_set):
+        type_mapping = {
+            'add': 'A',
+            'delete': 'D',
+            'edit': 'M',
+        }
+        file_changes = []
+        # https://docs.microsoft.com/en-us/rest/api/vsts/git/commits/get%20changes#versioncontrolchangetype
+        for change in patch_set:
+            change_type = type_mapping.get(change['changeType'])
+
+            if(change_type and change['item'] and change['item']['gitObjectType'] == 'blob'):
```

Comment:
should the second one be `change.get('item')`?

Reply:
good call